### PR TITLE
Update README.md to append a translation in Simplified Chinese

### PR DESCRIPTION
**Pull request by Higurashi-kagome**
Just added the README-zh-cn.md——a translation file of  Chinese.

### CODE REVIEW

**Comment by Sen-real** [comment]
Hello @Higurashi-kagome, to get your PR merged, please sign the following file (It's once only), and send it to lan.dong@autodesk.com. Thanks a lot!
https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/main/Autodesk_Form_Ind_Contrib_Agmt_for_Open_%20Source.pdf

**Comment by Higurashi-kagome** [comment]
Done. @Sen-real

File: README-zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+## 注意
+
+1. 其他 AutoLISP 扩展可能与此扩展冲突。如果无法在 LSP 文件中设置断点或无法使用启动/附着调试配置，则应禁用或卸载所有其他 AutoLISP 扩展。
+2. 在 Mac^®^ OS 上使用此扩展时，扩展设置中的“Launch Program”应为“../AutoCAD\<release\>.app/Contents/MacOS/AutoCAD”，而不是 AutoCAD\<release\>.app 的绝对路径。
+3. 对于 AutoCAD 专用工具集或基于 OEM 的产品，产品名称可能与 AutoCAD 或 acad.exe 不同。要将此扩展用于这些产品，请根据要调试的产品在扩展设置中设置对应“Launch Program”的值。
+4. 如果以管理员身份启动 AutoCAD，则同样应以管理员身份启动 Microsoft Visual Studio Code。否则，Microsoft Visual Studio Code 将无法找到用于附加调试的 AutoCAD 进程。
+
+## 声明
+
+Visual Studio Code Extension for AutoCAD® AutoLISP © 2020 Autodesk, Inc. All rights reserved.
```
Comment (Sen-real) [inline README-zh-cn.md:98]:
```suggestion
Visual Studio Code Extension for AutoCAD® AutoLISP © 版权所有 2020 Autodesk Inc. 保留所有权利。
```

File: README-zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### 商标
+
+本许可证未授予使用 Autodesk 的商品名、商标、服务标志或产品名称的权限，除非在描述作品来源和复制任何通知文件内容时出于合理和习惯的需要。Autodesk、Autodesk 徽标、AutoCAD 和AutoLISP 是 Autodesk，Inc. 和/或其子公司和/或附属公司在美国和/或其他国家/地区的注册商标。所有其他品牌名称、产品名称或商标均属于其各自的持有人。Autodesk 不对本文档中可能出现的印刷或图形错误负责。
+
+所有其他品牌名称、产品名称或商标均属于其各自的持有人。
+
+### 第三方商标等
+
+[NOTICE.md](https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/main/NOTICE.md)。
```
Comment (Sen-real) [inline README-zh-cn.md:112]:
```suggestion
### Third-Party Trademarks, Software Credits and Attributions

See [here](NOTICE.md).
```

Comment (Sen-real) [reply README-zh-cn.md:112]: Let's keep it in English if we have no idea of a proper translation.

File: README-zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+### 隐私
+
+要了解有关 Autodesk 联机和脱机隐私文档的详细信息，请参见 [Autodesk 隐私声明](https://www.autodesk.com/company/legal-notices-trademarks/privacy-statement)。
+
+### 商标
+
+本许可证未授予使用 Autodesk 的商品名、商标、服务标志或产品名称的权限，除非在描述作品来源和复制任何通知文件内容时出于合理和习惯的需要。Autodesk、Autodesk 徽标、AutoCAD 和AutoLISP 是 Autodesk，Inc. 和/或其子公司和/或附属公司在美国和/或其他国家/地区的注册商标。所有其他品牌名称、产品名称或商标均属于其各自的持有人。Autodesk 不对本文档中可能出现的印刷或图形错误负责。
+
+所有其他品牌名称、产品名称或商标均属于其各自的持有人。
```
Comment (Sen-real) [inline README-zh-cn.md:108]:
```suggestion
### Trademarks
The license does not grant permission to use the trade names, trademarks, service marks, or product names of Autodesk, except as required for reasonable and customary use in describing the origin of the work and reproducing the content of any notice file. Autodesk, the Autodesk logo, AutoCAD and AutoLISP are registered trademarks or trademarks of Autodesk, Inc., and/or its subsidiaries and/or affiliates in the USA and/or other countries. All other brand names, product names, or trademarks belong to their respective holders. Autodesk is not responsible for typographical or graphical errors that may appear in this document.

All other brand names, product names or trademarks belong to their respective holders.
```

Comment (Sen-real) [reply README-zh-cn.md:108]: We're programmers. Let's not touch it.

**Comment by Higurashi-kagome** [comment]
Thanks for your detailed and professional suggestions. @Sen-real

**Comment by Sen-real** [comment]
> Thanks for your detailed and professional suggestions. @Sen-real

Welcome. Just to double check with you: when viewing source code in VS Code, the links to readme.md and readme-zh-cn.md go to each other well. But after making and installing a real .vsix in VS Code, clicking the link will open a browser and navigate to github.

E.g. if I click the link to readme-zh-cn.md, the browser opens and navigates to https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/main/README-zh-cn.md (this URL doesn't exist because this PR is not merged yet).

Is this the expected behavior?

**Comment by Higurashi-kagome** [comment]
It's the expected behavior for me. The link will work fine I guess, just like we click the link to NOTICE.md, the browser opens and navigates to [HEAD/NOTICE.md](https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/HEAD/NOTICE.md).

**Comment by Sen-real** [comment]
> It's the expected behavior for me. The link will work fine I guess, just like we click the link to NOTICE.md, the browser opens and navigates to [HEAD/NOTICE.md](https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/HEAD/NOTICE.md).

@Higurashi-kagome 
We had a discussion on this. Could you please just append the Chinese translation into the same READEM.md?

**Comment by Higurashi-kagome** [comment]
> > It's the expected behavior for me. The link will work fine I guess, just like we click the link to NOTICE.md, the browser opens and navigates to [HEAD/NOTICE.md](https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/HEAD/NOTICE.md?rgh-link-date=2021-11-18T02%3A42%3A58Z).
> 
> @Higurashi-kagome We had a discussion on this. Could you please just append the Chinese translation into the same READEM.md?

Updated. The current links can't jump to each other in VSCode but work fine on GitHub, so, I guess, it will also work just like what we want on the Visual Studio Marketplace.

**Comment by Sen-real** [comment]
> > > It's the expected behavior for me. The link will work fine I guess, just like we click the link to NOTICE.md, the browser opens and navigates to [HEAD/NOTICE.md](https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/HEAD/NOTICE.md?rgh-link-date=2021-11-18T02%3A42%3A58Z).
> > 
> > 
> > @Higurashi-kagome We had a discussion on this. Could you please just append the Chinese translation into the same READEM.md?
> 
> Updated. The current links can't jump to each other in VSCode but work fine on GitHub, so, I guess, it will also work just like what we want on the Visual Studio Marketplace.

Hi @Higurashi-kagome,
I tried your latest code, and the links couldn't jump to each other. I think VS Code does have this capability, and the links can't jump simply because the link ids are wrong.

E.g. I locally added two headers for testing purpose:
Screenshot 2021-11-26 at 4 54 26 PM
Screenshot 2021-11-26 at 4 54 58 PM

I then:
- removed the extension from VS Code
- built the extension
- installed it in VS Code
- clicked the extension on "Extensions" panel to open the readme and check the result

They did jump as expected - they didn't navigate to github or open a browser; they just jumped inside VS Code.

You could build the extension on your own and have a try. Here're the steps:
https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/main/DevREADME.md#how-to-setup-the-dev-env-and-compile-the-code

After the 3rd step (`python pack.py`) is done, if there's nothing wrong you will see following messages:
image

**Comment by Higurashi-kagome** [comment]
Updated and tested. Works fine on my machine. It would be good if you can also test it. @Sen-real

**Comment by Sen-real** [comment]
> Updated and tested. Works fine on my machine. It would be good if you can also test it. @Sen-real
> 
> By the way, the links can't work as expected on the GitHub readme page now. This might be fixed by adding some code in pack.py. For example, make sure the links work well on GitHub firstly, and before generating the .vsix file, change the link and recover it after generating.

Yes it will work around the problem in this way, and we can do it if it's indeed a limitation.
But I think we'd better at first do some research, to find out why it doesn't work on Github website, and see if it can be solved directly in a regular/official approach.

**Comment by Sen-real** [comment]
What's your idea, @Higurashi-kagome?

**Comment by Higurashi-kagome** [comment]
I agree, @Sen-real . I will try to find a better solution but there is something else I need to do, so I am going to come back after lunch.

**Comment by Sen-real** [comment]
Cool, thanks a lot, @Higurashi-kagome!

**Comment by Higurashi-kagome** [comment]
I updated the readme file and it works well on GitHub now. But there is something wrong when the packing script uses `vsce`, I'm not interested in this and maybe it's just some config issues of the npm on my computer, so could you please test the links on VSCode? @Sen-real

**Comment by Sen-real** [comment]
> I updated the readme file and it works well on GitHub now. But there is something wrong when the packing script uses `vsce`, I'm not interested in this and maybe it's just some config issues of the npm on my computer, so could you please test the links on VSCode? @Sen-real

@Higurashi-kagome 
I tried, and the links worked very well in VSCode. Thanks a lot!

File: package.json [hunk 4]
```diff
@@ -659,7 +659,7 @@
 		"fs-extra": "^8.1.0",
 		"jschardet": "2.1.1",
 		"os-locale": "5.0.0",
-		"vsce": "1.88.0",
+		"vsce": "^2.5.0",
```
Comment (Sen-real) [inline package.json:662]: @Higurashi-kagome 
may I ask what's the reason of this change?

Comment (Higurashi-kagome) [reply package.json:662]: Just a change when I try to pack and test it using `vsce`. Ignore it would be good. @Sen-real

**Review by Sen-real — APPROVED** [review]
